src/components/pill-selector-dropdown/PillSelectorDropdownField.js

```diff
@@ -0,0 +1,127 @@
+// @flow
+import * as React from 'react';
+import getProp from 'lodash/get';
+import isEqual from 'lodash/isEqual';
+
+import type { FieldProps } from 'formik';
+
+import DatalistItem from '../datalist-item';
+import PillSelectorDropdown from './PillSelectorDropdown';
+import type { Option, Pill } from './flowTypes';
+
+import './PillSelectorDropdown.scss';
+
+type Props = FieldProps & {
+    /** CSS class for the component */
+    className?: string,
+    /** If true, the input control is disabled so no more input can be made */
+    disabled?: boolean,
+    /** If true, user can add pills not included in selector options */
+    isCustomInputAllowed: boolean,
+    /** Input label */
+    label: React.Node,
+    /** Array or Immutable list with data for the selected options shown as pills */
+    options: Array<Option>,
```

Comment:
I am enforcing some specific option format now, unlike existing pill selector that works on arbitrary objects. Having arbitrary objects allows more pill customization but also forces the client using the pill selector to [write all this](https://github.com/box/box-ui-elements/blob/master/src/components/pill-selector-dropdown/README.md).

From what I see is most pill selector use case is [TEXT x] so keeping data structure same as other components like select / multi-select ==> { displayText: something, value: something }